### PR TITLE
adobe-acrobat-reader 24.001.20615

### PR DESCRIPTION
**Pull request by WangEdward**
Update livecheck & adobe-acrobat-reader 24.001.20615

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by bevanjkay** [comment]
Did you first try updating the macOS version query attribute in the previous livecheck?

**Comment by WangEdward** [comment]
> Did you first try updating the macOS version query attribute in the previous livecheck?

It won't work, the old livecheck returns 24.001.20604 (previous version). Changing the query parameters also did not work.

**Comment by bevanjkay** [comment]
I think we changed to the api because the other url sometimes points to partial updates only. Which are patched inside the application and don't provide a full version. We can check, but maybe wait 24 hours and see if the previous url changes.

**Review by fxcoudert — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Agree with @bevanjkay. This is marked as an [optional update](https://helpx.adobe.com/acrobat/release-note/release-notes-acrobat-reader.html); we should see how long it takes for it to propagate to the api.

**Comment by krehel** [comment]
I was able to get this version just now (15:20p US Eastern) so it took some time to propagate, but is there on the api.

File: Casks/a/adobe-acrobat-reader.rb [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
   desc "View, print, and comment on PDF documents"
   homepage "https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html"
 
   livecheck do
-    url "https://rdc.adobe.io/reader/products?lang=en&site=landing&os=Mac%20OS%2010.15&api_key=dc-get-adobereader-cdn"
-    strategy :json do |json|
-      json.dig("products", "reader").map { |product| product["version"] }
-    end
+    url "https://armmf.adobe.com/arm-manifests/mac/AcrobatDC/reader/current_version.txt"
+    regex(/(\d+(?:\.\d+)+)/i)
```
Comment (razvanazamfirei) [inline Casks/a/adobe-acrobat-reader.rb:12]:
```suggestion
    url "https://rdc.adobe.io/reader/products?lang=en&site=landing&os=Mac%20OS%2010.15&api_key=dc-get-adobereader-cdn"
    strategy :json do |json|
      json.dig("products", "reader").map { |product| product["version"] }
    end
```

**Review by razvanazamfirei — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**